### ISSUE TITLE
Rivian Charger Sensors only on main driver login?

### ISSUE THREAD

**Issue by poldim** [bretterer/home-assistant-rivian#103]
Is there a way to get these stats to show up for secondary accounts?  Or should I be logging into the integration with my main driver credentials?

I initially (for a day or so) had the integration logged in with my main user profile until I read a [post](https://old.reddit.com/r/Rivian/comments/13e7bjo/rivian_integration_into_home_assistant/jjos01q/) recommending using a different driver account due to the risk of lockout.  I know @tmack8001 said some of those issues were corrected but I figured better safe than sorry.  

**Or is there another local way to get the sensor values?**  For reference, the charger identifies itself as _LCET-EVSE_ and the manufacturer is _u-blox AG_ according to the mac which appears to have made the networking chipset.  The box it came in shipped from 
Lite-On Technology
Corporation_Dayuan Fa
No.7, Ln.3, San Ho Road, Da-Yuan
Taoyuan City, 33754
(68496 COO: TW

**Comment by tmack8001** [comment]
Charger data no is not available from secondary accounts as unlike the vehicles chargers are not "shareable" to other Rivian accounts.

This is not a limitation of the HA component, but rather from Rivian itself.

**Comment by bplein** [comment]
I just stumbled upon this limitation (which is obvious to me in retrospect) today. I finally got off my tail and switched my HA integration instance to use a secondary driver account, got it all set up and moved over, and now my EVSE info is missing. I'd like it on my phone and in HA... but I understand it's the limitation on Rivian's side (single account for EVSE, multiple drivers for truck). 

Not having it on my phone may not be a big deal, unless something goes funky with the EVSE itself.
